### PR TITLE
Added some missing Shadow equipments

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

https://www.divine-pride.net/database/item/24687
https://www.divine-pride.net/database/item/24688
https://www.divine-pride.net/database/item/24689
https://www.divine-pride.net/database/item/24690
https://www.divine-pride.net/database/item/24691
https://www.divine-pride.net/database/item/24692
https://www.divine-pride.net/database/item/24693
https://www.divine-pride.net/database/item/24694
https://www.divine-pride.net/database/item/24695
https://www.divine-pride.net/database/item/24696
https://www.divine-pride.net/database/item/24697
https://www.divine-pride.net/database/item/24698
https://www.divine-pride.net/database/item/24699
https://www.divine-pride.net/database/item/24700
https://www.divine-pride.net/database/item/24701
https://www.divine-pride.net/database/item/24702
https://www.divine-pride.net/database/item/24703
https://www.divine-pride.net/database/item/24704
https://www.divine-pride.net/database/item/24705
https://www.divine-pride.net/database/item/24706
https://www.divine-pride.net/database/item/24707
https://www.divine-pride.net/database/item/24708
https://www.divine-pride.net/database/item/24709
https://www.divine-pride.net/database/item/24710
https://www.divine-pride.net/database/item/24711
https://www.divine-pride.net/database/item/24712
https://www.divine-pride.net/database/item/24713
https://www.divine-pride.net/database/item/24714
https://www.divine-pride.net/database/item/24715

### CODE REVIEW

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -2253,6 +2253,25 @@
 24677:24678,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L); bonus bMatkRate,2; if (.@sum>=18) { bonus bUseSPrate,40; bonus bNoGemStone; if (.@sum>=20) { bonus2 bMagicAddSize,Size_All,12; } } }
 24679:24680,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ARMOR)+getequiprefinerycnt(EQI_SHADOW_SHOES); bonus2 bAddClass,Class_All,2; if (.@sum>=18) { bonus bNoKnockback; if (.@sum>=20) { bonus2 bAddEle,Ele_All,12; } } }
 24681:24682,{ .@sum = getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L); bonus2 bAddClass,Class_All,2; if (.@sum>=18) { bonus bNoKnockback; if (.@sum>=20) { bonus2 bAddEle,Ele_All,12; } } }
+24687:24688:24689:24690:24691:24692,{ bonus bAspdRate,2; bonus bVariableCastrate,10; bonus2 bIgnoreDefRaceRate,RC_All,50; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-50; bonus2 bIgnoreMdefRaceRate,RC_All,50; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-50; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2256]:
```suggestion
24687:24688:24689:24690:24691:24692,{ bonus bAspd,2; bonus bVariableCastrate,-10; bonus2 bIgnoreDefRaceRate,RC_All,50; bonus2 bIgnoreDefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreDefRaceRate,RC_Player_Doram,-50; bonus2 bIgnoreMdefRaceRate,RC_All,50; bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-50; bonus2 bIgnoreMdefRaceRate,RC_Player_Doram,-50; }
```
For future reference I find it more practical to write the ID of the item where the combo description is put on in first

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -107125,6 +107125,647 @@ Body:
...
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      bonus bMaxHP,100;
+      bonus bVariableCastrate,5;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:107168]: Thanks

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -107125,6 +107125,647 @@ Body:
...
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      bonus bAtk,5;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:107243]: Thanks

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**